libffi/funcall.lisp

```diff
@@ -121,4 +120,6 @@
 
 (setf *foreign-structures-by-value* 'foreign-funcall-form/fsbv-with-libffi)
 
+(pushnew 'cffi-features:struct-by-value *features*)
```

Comment:
i'm just keeping the old code alive here, and apparently i was wrong assuming that cffi-features is the way to go... :)

:cffi-libffi is not a good symbol, though, because this is about the ability to pass structures by value, and using libffi is just one way to implement that.

maybe :cffi/struct-by-value then?

or :cffi-fsbv? (i never liked fsbv because for me it's a pain to type and remember, and doesn't communicate anything to the person first approaching the codebase)

but i'm not using this at all... i rely on explicit dependencies, so as for me i'm fine with the complete removal of this from *features*. it will just lead to headaches anyways if someone compiles their code with cffi-libffi loaded, and then tries to reload the fasls later on into an image without cffi-libffi... the only sane way to deal with this (that i know of) is explicit ASDF dependencies.